pywbemtools/pywbemcli/_cmd_connection.py

```diff
@@ -233,8 +233,13 @@ def connection_save(context, name):
 
 @connection_group.command('list', cls=PywbemcliCommand,
                           options_metavar=CMD_OPTS_TXT)
+@click.option('-f', '--full', is_flag=True,
+              default=False,
+              help='If set, display the full table. Otherwise display '
+                   ' a brief view(name, server, mock_server columns). '
```

Comment:
Two spaces in sentence (between "display" and "a").

Reply:
Fixed. DONE